### PR TITLE
Add Districts MX

### PR DESCRIPTION
**Pull request by RodrigoRibeiroSantos**
Including missing districts from Mexico

### CODE REVIEW

**Review by evannjw — APPROVED** [review]

File: identifiers/country-mx/districts.csv
Comment (jloutsenhizer) [inline identifiers/country-mx/districts.csv:1]: My understanding is that There are 300 federal electoral districts in Mexico: https://pdba.georgetown.edu/ElecSys/Mexico/mexico.html

However, we have 307 after this change. Based on [this wiki page](https://en.wikipedia.org/wiki/Federal_electoral_districts_of_Mexico), my understanding is that redistricting last happened last year for the upcoming election next year. Could you update these with validFrom (for new districts) and validThrough (for removed districts) to reflect this?

**Review by jloutsenhizer — CHANGES_REQUESTED** [review]
Overall looks good, I think we just need some `validFrom` and `validThrough` attributes to make the data a little clearer.

File: identifiers/country-mx/districts.csv
Comment (RodrigoRibeiroSantos) [reply identifiers/country-mx/districts.csv:1]: Added validFrom "2022-12-12" for new 7 districts.

Comment (jloutsenhizer) [reply identifiers/country-mx/districts.csv:1]: What's the significance of `2022-12-12` ? it looks like the next election is `2024-06-02`: https://en.wikipedia.org/wiki/2024_Mexican_general_election

I'd recommend using `2024-06-02` as the `validFrom` date, can you also add `validThrough` of `2024-06-02` for the 7 districts that are going away as well?

Comment (RodrigoRibeiroSantos) [reply identifiers/country-mx/districts.csv:1]: Changed the validFrom and validThrough to "2024-06-02". Thanks

Comment (jloutsenhizer) [reply identifiers/country-mx/districts.csv:1]: Sorry I think there's was a misunderstanding. The updates as is means each of these new districts is valid only on `2024-06-02`.

For `validThrough` I was requesting it be added to the 7 districts that are going away since we should have 300 valid districts on any given date.

**Comment by jloutsenhizer** [comment]
@RodrigoRibeiroSantos are there any updates on this?

**Comment by RodrigoRibeiroSantos** [comment]
> @RodrigoRibeiroSantos are there any updates on this?

Hi, we are finishing checking the removed districts. I will update the PR until this week.

**Comment by RodrigoRibeiroSantos** [comment]
> @RodrigoRibeiroSantos are there any updates on this?

@jloutsenhizer Hi, I just updated the districts removed. Thanks

File: identifiers/country-mx/districts.csv
Comment (jloutsenhizer) [reply identifiers/country-mx/districts.csv:1]: This is very close, I think the only part that still needs fixed is the new districts still have the `2024-06-02` date for both `validFrom` and `validThrough`. Those ones just need to be updated so the `2024-06-02` date is only set for the `validFrom` date.

**Comment by jloutsenhizer** [comment]
I left one more comment to correct around `validThrough` on the new districts. Otherwise this is close. Once you fix that make sure you also run `complie.py` again so that `country-mx.csv` gets updated.

@jpmckinney FYI for second review

**Comment by jpmckinney** [comment]
I reviewed, and can approve once the error is corrected.

File: identifiers/country-mx.csv
Comment (jpmckinney) [inline identifiers/country-mx.csv:1]:
```suggestion
id,name,validFrom,validThrough

```

Comment (jpmckinney) [reply identifiers/country-mx.csv:1]: Maybe just run `./scripts/compile.py mx`

Comment (RodrigoRibeiroSantos) [reply identifiers/country-mx.csv]: Runned compile.py mx, that rewrite the file country-mx. If needs any changes, let me know.

image

**Review by jloutsenhizer — APPROVED** [review]
This PR LGTM

**Review by jpmckinney — APPROVED** [review]

**Pull request merged**
